packages/youtube_player_flutter/lib/src/player/youtube_player.dart

```diff
@@ -297,8 +297,9 @@ class _YoutubePlayerState extends State<YoutubePlayer> {
         children: [
           Transform.scale(
             scale: controller.value.isFullScreen
-                ? (1 / _aspectRatio * MediaQuery.of(context).size.width) /
-                    MediaQuery.of(context).size.height
+                ? 1
```

Comment:
`isFullScreen ? 1 : 1,`

Doesn't make much sense to me, better just use 1 directly 👍